var/spack/repos/builtin/packages/boxlib/package.py

```diff
@@ -36,12 +36,14 @@ class Boxlib(CMakePackage):
 
     depends_on('mpi')
 
+    variant('dims', default=3, description='Number of spatial dimensions')
```

Comment:
Based on mvapich example I saw, yes. At least once I switched to that, I got intended behavior. Is there a better approach?